invenio_rest/views.py

```diff
@@ -80,10 +87,25 @@ def make_response(self, *args, **kwargs):
         :raises :py:class:`werkzeug.exceptions.NotAcceptable`: if no media type
         matches current Accept header.
         """
-        best = request.accept_mimetypes.best_match(self.serializers.keys())
-        if best is None:
-            abort(406)
-        return self.serializers[best](*args, **kwargs)
+        # First search for a local serializer if one has been set
+        if self.method_serializers:
```

Comment:
Yes: global serializer handling `application/json` and `application/marcxml+xml` with a method serializer overriding only `application/json`. But I don't think that this use case will be needed very often. In fact it could even create nasty bugs if you really wanted to disable `application/marcxml+xml` for the method. So your version is better.
If somebody wants to override part of the serializers for a method, they can just copy the ones in common from global serializer dict to the method one.